src/golang/lib/airflow/schedule.go

```diff
@@ -0,0 +1,297 @@
+package airflow
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	operator_db "github.com/aqueducthq/aqueduct/lib/collections/operator"
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	"github.com/aqueducthq/aqueduct/lib/collections/workflow_dag"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/job"
+	"github.com/aqueducthq/aqueduct/lib/storage"
+	"github.com/aqueducthq/aqueduct/lib/vault"
+	"github.com/aqueducthq/aqueduct/lib/workflow/artifact"
+	"github.com/aqueducthq/aqueduct/lib/workflow/operator"
+	"github.com/aqueducthq/aqueduct/lib/workflow/operator/connector/auth"
+	"github.com/aqueducthq/aqueduct/lib/workflow/utils"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/google/uuid"
+)
+
+const (
+	pollCompileAirflowInterval = 500 * time.Millisecond
+	pollCompileAirflowTimeout  = 30 * time.Second
+)
+
+// ScheduleWorkflow schedules a workflow with an Airflow backend.
+// It returns the Airflow DAG Python spec file and an error, if any.
+//
+// Assumptions:
```

Comment:
I think we could remove these comments after @kenxu95 's refactor

Reply:
Yeah I can clean this up once we stop using DBWorkflowDag here. Might be a while in the future though so this is great for now.